geode-assembly/src/test/java/org/apache/geode/test/dunit/rules/gfsh/GfshRule.java

```diff
@@ -26,6 +26,8 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import javax.mail.Folder;
```

Comment:
Is this import used? It looks like just the import was added so it's probably unused. Delete it?